src/figuresClasses.ts

```diff
@@ -1,19 +1,104 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  color:Color;
```

Comment:
Add a space for better readability
```suggestion
  color: Color;
```
```suggestion
  color:Color;
```